pages/api/s3-upload.ts

```diff
@@ -0,0 +1,72 @@
+import { ReadStream } from "fs";
+import fs from "fs";
+import { v4 as uuid } from "uuid";
+import { UploadResult, UploadFailure, UploadSuccess } from "../../lib/types";
+import { S3Client, PutObjectCommand, GetObjectCommand } from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import formidable from "formidable";
+
+const s3Client = new S3Client({ region: process.env.AWS_BUCKET_REGION }); // could be hard coded
```

Comment:
If we're not going to hard code it than let's set a default in case the env var is missing.  If we were to merge this PR the functionality wouldn't actually work on Staging because we would also need to do a infrastructure release that injects an env into the fargate instances.
ex. `{region: process.env.AWS_BUCKET_REGION ?? "ca-central-1"}`

Reply:
oaky. Lets for now hard code it. And we'll refactor it once the infra is ready.